BearMyBanner/settings/IBMBSettingsExtension.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace BearMyBanner.Settings
+{
+    public static class IBMBSettingsExtension
+    {
+        public static IBMBSettings SetDefaults(this IBMBSettings settings)
```

Comment:
Extension method (notice this in method parameters list). We can do them even for interfaces (that's how linq works). This way we can apply default values to different implementations. In this case for ModLib and test settings.